modules/custom/az_person/config/install/core.entity_view_display.node.az_person.az_row.yml

```
@@ -141,6 +141,21 @@ third_party_settings:
         attributes: ''
         effect: none
         speed: fast
+    group_linked_image:
+      children:
+        - field_az_media_image
+      label: 'Linked image'
+      parent_name: group_col_image
+      region: content
+      weight: 23
+      format_type: link
+      format_settings:
+        classes: ''
```

Before converting AZ Person to use the Drupal UI, we added the `az-person-row-img` class to the link containing the image and then used [a rule in az_person.css](https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_person/css/az_person.css) to hide the contextual edit button. The pencil icon button for media doesn't work in this case since clicking it simply takes you to the full node.

If we want to keep that button hidden like it was before, we could add the `az-person-row-img` class to the field group (suggested below) or we could update the selector in az_person.css to replace `az-person-row-img` with `field-group-link`.

```suggestion
        classes: az-person-row-img
```

---

Could also maybe use `az-person-image-link` as used on the card display.